src/main/scala/ru/org/codingteam/keter/scenes/game/GameMapView.scala

```diff
@@ -5,17 +5,15 @@ import ru.org.codingteam.keter.game.objects._
 import ru.org.codingteam.keter.map.TraverseUtils.{BoardCell, BoardCoords}
 import ru.org.codingteam.keter.map.{Surface, TraverseUtils}
 import ru.org.codingteam.keter.ui.IView
+import ru.org.codingteam.keter.ui.shape.Rectangle
 import ru.org.codingteam.keter.util.Logging
 import ru.org.codingteam.rotjs.interface.Display
 import ru.org.codingteam.rotjs.wrappers._
 
-class GameMapView(parent: GameScene, width: Int, height: Int, viewModel: GameMapViewModel) extends IView with Logging {
+class GameMapView(shape: Rectangle, parent: GameScene, viewModel: GameMapViewModel) extends IView with Logging {
```

Comment:
> `shape: Rectangle`

Why is the name of the variable more general than its type? It looks kind of strange ._.